hotfixes/4.3.15.cfg

```diff
@@ -0,0 +1,12 @@
+[buildout]
+
+hotfix-eggs =
+
```

Comment:
@shylux I've updated the plone-hotfixes constraints.
Could you give it a shot and run `hotfixes/update.py` on your branch?
I expect that it does not change a lot.